modules/openshift-developer-cli-debugging-an-application.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/debugging-applications-in-odo.adoc
+
+[id="debugging-an-application_{context}"]
+
+= Debugging an application
+
```

Comment:
Suggest adding a brief introductory sentence in this and the other module. Something like "You can debug an application using `odo` ..."